src/riot-rs-runqueue/src/runqueue.rs

```diff
@@ -129,6 +140,7 @@ mod clist {
             self.tail[rq as usize] == Self::sentinel()
         }
 
+        #[allow(clippy::missing_panics_doc, reason = "only used internally")]
```

Comment:
I don't think that fact that something is only used internally is a good reason to not add panic documentation. 
We also have them on other internal methods. Was there a specific reason for not adding something here?

Reply:
```suggestion
        /// # Panics
        /// 
        /// Panics if `n` >= `N_THREADS` or `rq` >= `N_QUEUES`. 
```

Should probably also be added on `Runqueue::add`, which is public.

> especially if that would help document `sentinel()`

`sentinel` is the largest value that a `u8` can represent (=255). Right now we simply rely on the fact that `N_THREADS` < 255 throughout the whole module (independently of this method here). This is included in the _Assumptions_ list in the `RunQueue` docs.